### PR TITLE
Merge request CSCFC4EMSCR-470 Add namespace as an editable field for schemas to develop branch

### PR DESCRIPTION
**Pull request by masillan**
Merge request to develop branch. CSCFC4EMSCR-470 Add namespace as an editable field for schemas implementation. Namespace can now be edited and changed.

### CODE REVIEW

File: package-lock.json
Comment (maariaw) [inline package-lock.json:1]: Should leave out these changes to package-lock.json. It's weird that it isn't in the gitignore, but we can discuss that separately.

**Review by maariaw — CHANGES_REQUESTED** [review]
Need to remove the changes to package-lock. Otherwise all clear.

File: package-lock.json
Comment (masillan) [reply package-lock.json:1]: Nice to have a comment. I had a monologue on this matter on MSCR Rocket Chat channel.

**Review by maariaw — APPROVED** [review]
OK

**Pull request merged**